monkey/monkey_island/cc/resources/agent_events.py

```diff
@@ -85,10 +89,21 @@ def _parse_event_filter_args(self) -> Tuple[Optional[Type[AbstractAgentEvent]],
                 f'Invalid value for success "{success_arg}", expected "true" or "false"'
             )
 
-        return type_, success
+        if timestamp_arg:
+            operator, timestamp = timestamp_arg.split(":")
+            if not operator or not timestamp or operator not in ("gt", "lt"):
+                raise Exception(f'Invalid timestamp argument "{timestamp_arg}"')
```

Comment:
We can improve the specificity with something like:

```suggestion
                raise Exception(f'Invalid timestamp argument "{timestamp_arg}: no operator provided"')
```